app/helpers/middleware_jms_helper/textual_summary.rb

```diff
@@ -0,0 +1,46 @@
+module MiddlewareJmsHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    %i(name nativeid driver_name jndi_name connection_url enabled)
+  end
+
+  def textual_group_relationships
+    # Order of items should be from parent to child
+    %i(ems middleware_server)
+  end
+
+  def textual_group_smart_management
+    %i(tags)
+  end
+
+  def textual_name
```

Comment:
I don't think this is needed, check include TextualMixins::TextualName part in other similar places, I think it already does that